### ISSUE TITLE
App token login name does not match Issue

### ISSUE THREAD

**Issue by HyCore** [nextcloud/talk-desktop#222]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Download any version of Nextcloud Talk for windows (not tested with other)
2. Launch it and try to connect with either email or username.

### Expected behaviour
Connect without error.

### Actual behaviour
Show this error
![image](https://github.com/nextcloud/talk-desktop/assets/48274082/b1333727-3b6b-4349-9115-0dab616fc239)

## Desktop client

**Talk Desktop client version:** v1.0.0

**Operating system:** Windows

**Operating system version:** Windows 11

**Microphone available:** yes

**Camera available:** yes

## Server

**Nextcloud version:** 27.0.0.5

**Talk app version:** 17.0.0

**Custom Signaling server configured:** no

**Custom TURN server configured:** yes

**Custom STUN server configured:** yes

## Logs
**Server Logs**:
```
{"reqId":"smH7kixcW3Zft3CZEC4e","level":3,"time":"June 13, 2023 12:22:25","remoteAddr":"REDACTED","user":"--","app":"no app in context","method":"GET","url":"/ocs/v2.php/cloud/user","message":"App token login name does not match","userAgent":"Mozilla/5.0 (Windows) Nextcloud-Talk v0.6.0","version":"27.0.0.5","data":{"tokenLoginName":"Paul Chauvat","sessionLoginName":"Paul+Chauvat"}}
{"reqId":"BwJ3R2Ux8vKFsMdeG1Cg","level":3,"time":"June 13, 2023 12:22:25","remoteAddr":"REDACTED","user":"--","app":"no app in context","method":"GET","url":"/ocs/v2.php/cloud/capabilities","message":"App token login name does not match","userAgent":"Mozilla/5.0 (Windows) Nextcloud-Talk v0.6.0","version":"27.0.0.5","data":{"tokenLoginName":"Paul Chauvat","sessionLoginName":"Paul+Chauvat"}}
{"reqId":"GI5STkaMX7YdLVd4Lnmf","level":3,"time":"June 13, 2023 12:28:51","remoteAddr":"REDACTED","user":"--","app":"no app in context","method":"GET","url":"/ocs/v2.php/cloud/capabilities","message":"App token login name does not match","userAgent":"Mozilla/5.0 (Windows) Nextcloud-Talk v0.6.0","version":"27.0.0.5","data":{"tokenLoginName":"Paul Chauvat","sessionLoginName":"Paul+Chauvat"}}
{"reqId":"ArtnCTxOindtG19vOYwm","level":3,"time":"June 13, 2023 12:28:51","remoteAddr":"REDACTED","user":"--","app":"no app in context","method":"GET","url":"/ocs/v2.php/cloud/user","message":"App token login name does not match","userAgent":"Mozilla/5.0 (Windows) Nextcloud-Talk v0.6.0","version":"27.0.0.5","data":{"tokenLoginName":"Paul Chauvat","sessionLoginName":"Paul+Chauvat"}}
```

**Talk Dev Log**:
```
host.chauvat.com/ocs/v2.php/cloud/user:1     Failed to load resource: the server responded with a status of 401 ()
index.js:2 ocode: "ERR_BAD_REQUEST"config: adapter: ƒ (A)data: undefinedenv: {FormData: null}headers: {Accept: 'application/json, text/plain, */*', OCS-APIRequest: 'true'}maxBodyLength: -1maxContentLength: -1method: "get"timeout: 0transformRequest: [ƒ]transformResponse: [ƒ]transitional: {silentJSONParsing: true, forcedJSONParsing: true, clarifyTimeoutError: false}url: "https://host.chauvat.com/ocs/v2.php/cloud/user"validateStatus: ƒ (A)withCredentials: truexsrfCookieName: "XSRF-TOKEN"xsrfHeaderName: "X-XSRF-TOKEN"[[Prototype]]: Objectmessage: "Request failed with status code 401"name: "AxiosError"request: XMLHttpRequest {onreadystatechange: null, readyState: 4, timeout: 0, withCredentials: true, upload: XMLHttpRequestUpload, …}onabort: ƒ ()onerror: ƒ ()onload: nullonloadend: ƒ E()onloadstart: nullonprogress: nullonreadystatechange: nullontimeout: ƒ ()readyState: 4response: "<?xml version=\"1.0\"?>\n<ocs>\n <meta>\n  <status>failure</status>\n  <statuscode>997</statuscode>\n  <message>Unauthorised</message>\n </meta>\n <data/>\n</ocs>\n"responseText: "<?xml version=\"1.0\"?>\n<ocs>\n <meta>\n  <status>failure</status>\n  <statuscode>997</statuscode>\n  <message>Unauthorised</message>\n </meta>\n <data/>\n</ocs>\n"responseType: ""responseURL: "https://host.chauvat.com/ocs/v2.php/cloud/user"responseXML: documentstatus: 401statusText: ""timeout: 0upload: XMLHttpRequestUpload {onloadstart: null, onprogress: null, onabort: null, onerror: null, onload: null, …}withCredentials: true[[Prototype]]: XMLHttpRequestresponse: {data: '<?xml version="1.0"?>\n<ocs>\n <meta>\n  <status>fail…>Unauthorised</message>\n </meta>\n <data/>\n</ocs>\n', status: 401, statusText: '', headers: {…}, config: {…}, …}config: {transitional: {…}, transformRequest: Array(1), transformResponse: Array(1), timeout: 0, adapter: ƒ, …}data: "<?xml version=\"1.0\"?>\n<ocs>\n <meta>\n  <status>failure</status>\n  <statuscode>997</statuscode>\n  <message>Unauthorised</message>\n </meta>\n <data/>\n</ocs>\n"headers: {access-control-allow-credentials: 'true', access-control-allow-headers: 'Authorization, OCS-APIRequest, Content-Type, If-None-Match, Depth', access-control-allow-methods: 'GET, POST, PUT, PATCH, DELETE, PROPFIND, MKCOL', access-control-allow-origin: 'file://', access-control-expose-headers: 'ETag, x-nextcloud-talk-modified-before, x-nextclou…tuses, x-chat-last-given, x-chat-last-common-read', …}request: XMLHttpRequest {onreadystatechange: null, readyState: 4, timeout: 0, withCredentials: true, upload: XMLHttpRequestUpload, …}status: 401statusText: ""[[Prototype]]: Object[[Prototype]]: Errorconstructor: ƒ o(A,t,i,a,o)toJSON: ƒ ()isAxiosError: true[[Prototype]]: Object
login @ index.js:2
host.chauvat.com/ocs/v2.php/cloud/capabilities:1     Failed to load resource: the server responded with a status of 401 ()
DevTools failed to load source map: Could not load content for file:///C:/Users/chauvat/Downloads/Talk/resources/app/.webpack/renderer/authentication_window/preload.js.map: Unexpected end of JSON input
DevTools failed to load source map: Could not load content for file:///C:/Users/chauvat/Downloads/Talk/resources/app/.webpack/renderer/authentication_window/index.js.map: Unexpected end of JSON input


</details>

**Comment by ShGKme** [comment]
If a username contains a space, then it was fixed recently in https://github.com/nextcloud/talk-desktop/pull/202

A new version with this update is to be released soon this week.

**Comment by HyCore** [comment]
i hope so, as of rn it's still not the case :/

**Comment by ShGKme** [comment]
Hi @HyCore 

I'm sorry for a long release this time. `0.7.0` is released now and you can try it:
https://github.com/nextcloud-releases/talk-desktop/releases/tag/v0.7.0

Is the issue solved for you with 0.7.0?

**Comment by HyCore** [comment]
Yes it fixed it ;)
